cfme/test_requirements.py

```diff
@@ -43,7 +43,7 @@ def test_quota_alert():
             "assignee_id": "mkourim",
             "category_ids": "category_id1, category_id2",
             "due_date": "2018-09-30",
-            "planned_in_ids": "planned_id1, planned_id2",
+            "planned_in_ids": "planned_id1 planned_id2",
```

Comment:
removed comma on purpose?

Reply:
@mshriver, do we use different separators in different fields ?  look at category_ids above. it has comma separator

Reply:
Not sure about category_ids specifically, and if you read the comment in its entirety you'll see the message from Martin that this is potentially an inaccurate example.

In testing, polarion xunit import accepts space delineated planned in ID's (which is different from the displayed planned-in value). Comma separated resulted in xunit import failures.